Makefile

```diff
@@ -15,9 +14,27 @@ else
 	CHECK  ?= which
 endif
 
+ifeq ($(COMMIT),)
+  COMMIT := $(shell git rev-parse --short HEAD 2> /dev/null)
+endif
+
+ifeq ($(BUILDTIME),)
+  BUILDTIME := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ" 2> /dev/null)
+endif
+ifeq ($(BUILDTIME),)
```

Comment:
Why would the previous setting of BUILDTIME fail? Also, why not have a fallback for the other variables?

Reply:
I got this snippet from the makefile that was shared in the issue description and there buildtime had a fallback value. We can add similar logic to all values but even I felt like it wasn't necessary because the version command also does the same and sets it to unknyif the value is missing.